promises.cf

```diff
@@ -57,6 +57,10 @@ body common control
 
       version => "CFEngine Promises.cf 3.7.0";
 
+      cfengine_3_7.(debian||redhat)::
+          package_inventory => { $(package_module_knowledge.platform_default) };
+          package_module => $(package_module_knowledge.platform_default);
```

Comment:
> Do these have sensible defaults inside of the C code?

Not ATM, the default is nothing. We could add it, but it's still not ideal. It means that no one can add a package inventory module besides the two standard ones as long as they have 3.6 hosts.